Mathlib/CategoryTheory/Category/ReflQuiv.lean

```diff
@@ -70,11 +70,37 @@ theorem forgetToQuiv_faithful {V W : ReflQuiv} (F G : V ⥤rq W)
     (hyp : forgetToQuiv.map F = forgetToQuiv.map G) : F = G := by
   cases F; cases G; cases hyp; rfl
 
-theorem forgetToQuiv.Faithful : Functor.Faithful (forgetToQuiv) where
+instance forgetToQuiv.Faithful : Functor.Faithful (forgetToQuiv) where
   map_injective := fun hyp ↦ forgetToQuiv_faithful _ _ hyp
 
 theorem forget_forgetToQuiv : forget ⋙ forgetToQuiv = Quiv.forget := rfl
 
+/-- An isomorphism of quivers lifts to an isomorphism of reflexive quivers given a suitable
+compatibility with the identities. -/
+def isoOfQuivIso {V W : Type u} [ReflQuiver V] [ReflQuiver W]
+    (e : Quiv.of V ≅ Quiv.of W)
+    (h_id : ∀ (X : V), e.hom.map (𝟙rq X) = ReflQuiver.id (obj := W) (e.hom.obj X)) :
+    ReflQuiv.of V ≅ ReflQuiv.of W where
+  hom := ReflPrefunctor.mk e.hom h_id
+  inv := ReflPrefunctor.mk e.inv
+    (fun Y => (Quiv.homEquivOfIso e).injective (by simp [Quiv.hom_map_inv_map_of_iso, h_id]))
+  hom_inv_id := by
+    apply forgetToQuiv.map_injective
+    exact e.hom_inv_id
+  inv_hom_id := by
+    apply forgetToQuiv.map_injective
+    exact e.inv_hom_id
+
+section
+variable {V W : Type u } [ReflQuiver V] [ReflQuiver W]
+  (e : V ≃ W) (he : ∀ (X Y : V), (X ⟶ Y) ≃ (e X ⟶ e Y))
+
+/-- Compatible equivalences of types and hom-types induce an isomorphism of quivers. -/
+def isoOfEquiv (h_id : ∀ (X : V), he _ _ (𝟙rq X) = ReflQuiver.id (obj := W) (e X)) :
+    ReflQuiv.of V ≅ ReflQuiv.of W := isoOfQuivIso (Quiv.isoOfEquiv e he) h_id
+
+end
```

Comment:
As there is only one declaration here, the variables should appear as `def isoOfEquiv {V W...`